<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="l_left_currency">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>61</y>
     <width>205</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="l_right_currency">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>205</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_left_currency">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>320</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt;</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_right_currency">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>320</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_search_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Строка поиска</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Имеющаяся валюта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>44</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавляемая валюта</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_save_and_exit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить и выйти</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_exit_without_save">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти без сохранения</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
